tests/functional/u/undefined/undefined_variable_py38.py

```diff
@@ -173,9 +173,9 @@ def expression_in_ternary_operator_inside_container_tuple():
     return [(val3, val3) if (val3 := 'something') else 'anything']
 
 
-def expression_in_ternary_operator_inside_container_wrong_position():
-    """2-element list where named expression comes too late"""
-    return [val3, val3 if (val3 := 'something') else 'anything']  # [used-before-assignment]
+def expression_in_ternary_operator_inside_container_later_position():
+    """Named expression follows unrelated item in container"""
+    return [23, val3 if (val3 := 'something') else 'anything']
```

Comment:
This PR trades fixing this false positive for not addressing the false negative previously tested here. I think this is both more important and more common.

Reply:
We should put that comment directly in the code imo

Reply:
3aa45afd2e46455269d4af77606c088b8298a15f